lib/container-info.js

```diff
@@ -1,20 +1,103 @@
-const getContainerInfo = require('container-info')
-
-/**
- * Function that uses `container-info`
- * package to extract container information
- * and apply additional naming rules
- *
- * Injectable getInfo for testing purposes
- *
- * https://github.com/elastic/apm/blob/main/specs/agents/metadata.md#containerkubernetes-metadata
- */
-module.exports = (getInfo = getContainerInfo) => {
-  const info = getInfo.sync()
-  if (!info) { return info }
-
-  if (info.podId) {
-    info.podId = info.podId.replace(/_/g, '-')
+'use strict'
```

Comment:
NOTE: Original is here if diffs are required: https://github.com/Qard/container-info/blob/v1.1.0/index.js